tamr_unify_client/models/dataset/usage.py

```diff
@@ -0,0 +1,39 @@
+from tamr_unify_client.models.dataset.use import DatasetUse
+
+
+class DatasetUsage:
+    """
+    The usage of a dataset and its downstream dependencies.
+
+    See https://docs.tamr.com//reference#retrieve-downstream-dataset-usage
```

Comment:
this is super nitpicky, but "See "https://docs.tamr.com//reference#retrieve-downstream-dataset-usage""

Reply:
Doesn't seem to make any difference in the docs